### PR TITLE
VS Code Quarkus 1.2.0 blog post

### PR DESCRIPTION
**Pull request by xorye**
Hi everyone, this is the rough draft for the VS Code Quarkus 1.2.0 blog post: https://docs.google.com/document/d/1qJVuN15Jdab1j6zELXG9Sn0ButXpDqPAAIkMxPi6Sfs/edit?usp=sharing
and I would like to get some feedback. Please feel free to add comments on the google doc, or on this PR.

Here is the video that accompanies it: https://youtu.be/BQTjtxZ3oqE

This PR does not contain the adoc for the blog post yet, which is why this is a draft PR. 

Thank you

### CODE REVIEW

File: _posts/2019-11-15-vscode-quarkus=1.2.0.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (gsmet) [inline _posts/2019-11-15-vscode-quarkus=1.2.0.adoc]: I don't think having an = in the URL is a good idea.

**Comment by gsmet** [comment]
I will wait for the asciidoc because I will need to check the formatting
too and I prefer to review everything in one pass.

**Comment by xorye** [comment]
@gsmet ok sounds good, I've pushed the asciidoc

**Comment by gsmet** [comment]
Ok cool, let me have a look.

**Comment by gsmet** [comment]
So, it's perfect and awesome, thanks a lot for that! Do you want it published now?

**Comment by gsmet** [comment]
@xorye OK, I added a note for that. And let's change the post date then too.

**Comment by maxandersen** [comment]
just renamed to WIP then the WIP bot will mark it as not completed in checks.

**Comment by gsmet** [comment]
@xorye in the end, we decided to push a CR2 today.

Final is planned for Monday if all goes according to plan.

**Comment by xorye** [comment]
Great, thanks for letting me know.

I've also made some small changes to the blog post and changed the date to November 21. I'm hoping we can merge this tomorrow (Nov. 21) right after the RH dev blog version is published, would that be alright? I will post a comment here when the RH Dev blog version is published (which will be published Nov.21).

**Comment by gsmet** [comment]
@xorye sure, ping me here when you want it merged.

File: _posts/2019-11-21-vscode-quarkus-1.2.0.adoc [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+To open the project generation wizard, open the VS Code command
+palette and call the `Quarkus: Generate a Quarkus project` command.
+
+image::01_gradle_project_gen.gif[alt="Generating a Gradle project"]
+
+=== Gradle - Debug Quarkus project
+The debug command will now detect whether your currently opened Quarkus
+project is a Maven or Gradle project, and will start the application by
+running the Quarkus Dev command
+(`mvn quarkus:dev` for Maven, `gradle quarkusDev` for Gradle).
```
Comment (gwenneg) [inline _posts/2019-11-21-vscode-quarkus-1.2.0.adoc]:
```suggestion
(`./mvnw quarkus:dev` for Maven, `./gradlew quarkusDev` for Gradle).
```

File: _posts/2019-11-21-vscode-quarkus-1.2.0.adoc [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+_both_ Gradle and Maven projects.
+
+== New `application.properties` features
+
+=== Quick Fix for unknown property name
+The validation support provided by Quarkus Tools for Visual Studio Code
+checked for unknown property keys in your `application.properties` file.
+This release brings a new Quick Fix that suggests known properties for your
+unknown properties.
+This feature is kind of like autocorrect or "spellcheck" for your property keys.
```
Comment (gwenneg) [inline _posts/2019-11-21-vscode-quarkus-1.2.0.adoc]:
```suggestion
This feature is kind of like autocorrect or "spellcheck" for your properties keys.
```
Or `property key` maybe.

File: _posts/2019-11-21-vscode-quarkus-1.2.0.adoc (hunk 2 above)
Comment (xorye) [reply _posts/2019-11-21-vscode-quarkus-1.2.0.adoc]: Thank you for the suggestion

File: _posts/2019-11-21-vscode-quarkus-1.2.0.adoc (hunk 3 above)
Comment (xorye) [reply _posts/2019-11-21-vscode-quarkus-1.2.0.adoc]: Thank you for the suggestion, but I think "property keys" makes the most sense here.

**Comment by xorye** [comment]
@gwenneg thank you!

@gsmet this can now be merged

**Comment by gsmet** [comment]
Done, it should be online in a few minutes.

Thanks!

**Pull request merged**